src/components/Sidebar/Sidebar.jsx

```diff
@@ -64,17 +75,26 @@ export const Sidebar = ({ logo, children, className, footer, notifications, isFi
         }
     };
 
+    const isStickyHeaderFooter = isStickyHeader && isStickyFooter;
+
     return (
         <div
             className={clsx(
+                sidebarScroll,
                 "ui-sidebar",
                 isFixed ? "fixed" : "relative",
-                "z-20 flex h-full w-16 flex-col overflow-y-auto bg-black py-2 px-1 text-white md:w-24 xl:w-50",
+                !isStickyHeaderFooter && "overflow-y-auto",
```

Comment:
py-2 px-1 why this not needed now

Reply:
this created a gap under the footer and above the header which resulted in an issue when menu items were visible after scrolling top or bottom